packages/app/pages/course/[cid]/queue/[qid].tsx

```diff
@@ -32,16 +32,17 @@ export default function Queue() {
   const role = useRoleInCourse(Number(cid));
 
```

Comment:
oh one more thing! can we prevent TAs from helping students if they're not checked in? we could just disable the student sidebar buttons for when they're not checked in